client/languages.ts

```diff
@@ -0,0 +1,23 @@
+// languages.ts
+
+// giving a name might help comprehension
+// I'm pretty sure that the `languages.js` file
+// is autogenerated otherwise we could include
+// the actual slugs for extra type safety - here
+// it's really not needed or helpful though
```

Comment:
these comments here and above should be removed from the final code.

Reply:
ahh sorry